prod10-sp2

david-prochazka opened the pull request:
Link to application: https://eso.vse.cz/~prod10/sp2/

david-prochazka committed:
prod10-sp2

Link to application: https://eso.vse.cz/~prod10/sp2/

david-prochazka committed:
Finální verze

Link to application: https://eso.vse.cz/~prod10/sp2/

nvbach91 commented on sp2/assets/js/main.js:
```diff
@@ -0,0 +1,406 @@
+    const baseApiUrl = 'https://api.spoonacular.com';
+    const apiKey = '?apiKey=c942e42ab9834c0497ff0de13bd6ac5d';
+
+    var ingredients = '';
```
let/const

nvbach91 commented on sp2/assets/js/main.js:
```diff
@@ -0,0 +1,406 @@
+        recipesCount = 0;
+
+        loadButton.empty();
+        spinner.appendTo($('#load-button'));
```
`loadButton`

nvbach91 reviewed (changes requested):
musite vytvorit novy PR se semestralkou ve slozce `/www/prod10/sp2`

david-prochazka commented:
Jak to prosím myslíte? Nerozumím tomu, myslel jsem, že jsem to tak dělal. Postupoval jsem podle návodu.

david-prochazka committed:
Finální verze s opravami

var jsem změnil na let
opravil jsem $('#load-button') na loadButton

přidal jsem k receptům možnost si zobrazit popis receptu

+ nová verze nahrána na eso: https://eso.vse.cz/~prod10/sp2/

david-prochazka committed:
Delete index.html

david-prochazka committed:
Delete main.js

david-prochazka committed:
Delete style.css

nvbach91 reviewed (approved):
![how-to-create-presentation-slides-that-are-out-of-this-world-by-slidecomet-itseugenec-kaixinspeaking-28-638](https://user-images.githubusercontent.com/20724910/104008962-8b66ea80-51aa-11eb-9b47-d64347f9797a.jpg)
